xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-fontawesome/src/main/resources/IconThemes/FontAwesome.xml

```diff
@@ -288,7 +288,8 @@ disk = floppy-o
 drink = glass
 drive = hdd-o
 email = envelope-o
-emoticon_smile=smile-o
+emoticon_smile= smile-o
+emoticon_unhappy = frown-o
```

Comment:
I think this change is unrelated to this PR.

Reply:
Removed in https://github.com/xwiki/xwiki-platform/pull/3023/commits/9e7759387dc99ee81b038d9712712fd62ab6ada7 :+1: